tests/aave/ethereum/aave_interests_test.sql

```diff
@@ -0,0 +1,22 @@
+with unit_test1
+    as (select case
+                 when variable_borrow_apy == 0.024106036652769853 then true
+                 else false
+               end as test
+        from   {{ ref('aave_v2_ethereum_interest_rates' )}}
+        where  reserve = '0xdac17f958d2ee523a2206206994597c13d831ec7'
+               and evt_block_time = '2022-08-22 12:00'),

```

Comment:
'evt_block_time' doesn't exist in your model, but rather 'hour' column that you built via date_trunc. this filter should be ' hour = '